package.json

```diff
@@ -87,6 +87,7 @@
     "country-telephone-data": "0.6.0",
     "exenv": "1.2.2",
     "focus-trap": "5.0.0",
+    "jest-styled-components": "6.3.3",
```

Comment:
You can rebase on the `styled-components` branche. I already imported it and setup the config